### PR TITLE
Check for high availability flag in env variables

### PR DESCRIPTION
**Pull request by CatBakun**
Inverted the connection method condition based on flag comming from proccess env.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ var events = require('events');
 var Q = require('q');
 var nunjucks = require('nunjucks');
 var util = require('./src/util/util');
-
+var highAvailability = process.env.HIG_AVAILABILITY;
```
Comment (travist) [inline index.js]: Can we rename this flag?... I would prefer if it were more descriptive of what is going on. Maybe ```MONGO_HIGH_AVAILABILITY``` or something like that would be good for me.

Comment (CatBakun) [inline index.js]: Done in 558e8b1

**Comment by travist** [comment]
I am going to bring this into a different PR so the tests and lint check out.

**Comment by travist** [comment]
pulling this into https://github.com/formio/formio/pull/252

**Pull request closed without merging**